e2e/_suites/ingest-manager/features/ingest-manager.feature

```diff
@@ -0,0 +1,46 @@
+@ingest
+Feature: Enable Fleet user and create initial Kibana setup
+
+Scenario: Enrolling an agent
+  Given there is a "Fleet" user in Kibana
+    And the "Fleet" Kibana setup has been created
+  When the agent binary is installed in the target host
+  Then the dashboards for the agent are present in Elasticsearch
+    And the agent shows up in Kibana
```

Comment:
Is it possible to get this without checking the UI, maybe an API call? I'd like to avoid any U/DOMI interaction if possible

Reply:
You can call `GET /api/ingest_manager/fleet/agents`